imagepipeline/src/main/java/com/facebook/imagepipeline/memory/BitmapCounter.java

```diff
@@ -59,7 +54,7 @@ public void release(Bitmap value) {
    * @param bitmap to include in the count
    * @return true if and only if bitmap is successfully included in the count
    */
-  public synchronized boolean increase(Bitmap bitmap) {
+  public synchronized boolean includeBitmap(Bitmap bitmap) {
```

Comment:
The old name is clearer IMO.

Reply:
Thank you for your notification. Do you think that, the other case 'generate' to 'createJpeg' improve the readability?